### PR TITLE
networkctl: tiny cleanups about alternative names

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: src/network/networkctl.c [hunk 1]
```diff
@@ -380,7 +380,7 @@ static int decode_link(sd_netlink_message *m, LinkInfo *info, char **patterns, b
                 return r;
 
         r = sd_netlink_message_read_strv(m, IFLA_PROP_LIST, IFLA_ALT_IFNAME, &altnames);
-        if (r < 0 && !IN_SET(r, -EOPNOTSUPP, -ENODATA))
+        if (r < 0 && r != -ENODATA)
```
Comment (keszybz) [inline src/network/networkctl.c:383]: The commit message is unclear. You say "this should just return -ENODATA" but the code change is to return -EOPNOTSUPP. I'm not sure if the code change is right or the commit message ;) Please make them agree.

Comment (yuwata) [reply src/network/networkctl.c:383]: Here, 'this' means `sd_netlink_message_read_strv()`. I've updated the commit message. I hope now it is clear.

Comment (keszybz) [reply src/network/networkctl.c:383]: Hmm, OK. But to nitpick: why "should"? I assume you mean "sd_netlink_message_read_strv() returns -ENODATA".

Comment (yuwata) [reply src/network/networkctl.c:383]: Ah, right...

**Pull request merged**